nautobot/dcim/models/sites.py

```diff
@@ -215,3 +215,10 @@ def to_csv(self):
             self.contact_email,
             self.comments,
         )
+
+    def clean_fields(self, exclude=None):
```

Comment:
```suggestion
    def clean_fields(self, exclude=None):
        """Explicitly convert latitude/longitude to strings to avoid floating-point precision errors."""
```